solo-chains/runtime/starlight/src/tests/integration_test.rs

```diff
@@ -276,3 +276,34 @@ fn test_configuration_on_session_change() {
         assert_eq!(CollatorConfiguration::config().collators_per_container, 10);
     });
 }
+
+#[test]
```

Comment:
Add more tests related to:

- creation of profile
- assignation to a chain
- root can assign to a chain forcedly
- on deregistration, what happens with assignation

Reply:
Those cases are covered by the TS test that I've added. Should I also add equivalents for integration tests?

Reply:
I would rather have duplicated versions, specially if those tests are already in dancebox as rust integration tests too

Reply:
Fixed